lib/translate/inquiry.cc

```diff
@@ -0,0 +1,298 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "inquiry.h"
+#include "string.h"
+namespace inquiry {
```

Comment:
Other translation PRs currently use a single namespace: "translator". We should standardize whether or not we would rather use individual namespaces for each command

Reply:
a single namespace would get very bloated. i think command-specific namespaces make more sense since each mini engine has a lot of functions that will only be internally called within that command-specific namespace.
plus this seems clean
`inquiry::translate()`
`read::translate()`

Reply:
Large numbers of namespaces isn't idiomatic. Internal-only functions can be defined in an anonymous namespace at the top of the `.cc` file and entirely absent from the header, provided that they're only needed within a single translation unit.